build.gradle.kts

```diff
@@ -4,11 +4,34 @@ plugins {
 }
 
 configurations {
+  implementation { exclude(module = "spring-boot-starter-web") }
+  implementation { exclude(module = "spring-boot-starter-tomcat") }
   testImplementation { exclude(group = "org.junit.vintage") }
 }
 
 dependencies {
+  implementation("org.springframework.boot:spring-boot-starter-data-r2dbc")
+  runtimeOnly("org.springframework.boot:spring-boot-starter-jdbc")
   implementation("org.springframework.boot:spring-boot-starter-webflux")
+  implementation("org.springframework.boot:spring-boot-starter-security")
+  implementation("org.springframework.boot:spring-boot-starter-oauth2-resource-server")
+
+  implementation("org.jetbrains.kotlinx:kotlinx-coroutines-core:1.6.4")
+  implementation("org.jetbrains.kotlinx:kotlinx-coroutines-reactor:1.6.4")
+
+  implementation("org.springdoc:springdoc-openapi-webflux-ui:1.6.14")
+  implementation("org.springdoc:springdoc-openapi-kotlin:1.6.14")
+  implementation("org.springdoc:springdoc-openapi-security:1.6.14")
+
+  implementation("io.opentelemetry:opentelemetry-api:1.22.0")
+
+  runtimeOnly("org.flywaydb:flyway-core")
+  runtimeOnly("org.postgresql:r2dbc-postgresql:0.9.3.RELEASE")
+  runtimeOnly("org.postgresql:postgresql:42.5.1")
+
+  testImplementation("org.jetbrains.kotlinx:kotlinx-coroutines-test:1.6.4")
+
+  testImplementation("com.github.tomakehurst:wiremock-standalone:2.27.2")
```

Comment:
Are we needing wiremock - I was thinking mock server would be OK?!

Reply:
I think I've added too much here, don't think we need this for now, was used with https://github.com/ministryofjustice/hmpps-staff-lookup-service/blob/d06d66478b3bfed6f9feb7afaf1cd6f6e594c8be/src/test/kotlin/uk/gov/justice/digital/hmpps/hmppsstafflookupservice/wiremock/HmppsAuthMockServer.kt but think that will be for clients

Reply:
Going to remove for now - we will use Mock Server